src/runtime/rule/dup.rs

```diff
@@ -82,7 +109,7 @@ pub fn apply(ctx: ReduceCtx) -> bool {
   // x <- N
   // y <- N
   // ~
-  else if get_tag(arg0) == U60 {
+  if get_tag(arg0) == U60 {
```

Comment:
What's the point of these changes? If you want to avoid triggering clippy::if_same_then_else you should just merge the U60 block and F60 block together. Better yet, use match instead of chaining if-else.